ykrt/src/compile/jitc_yk/jit_ir/parser.rs

```diff
@@ -128,6 +127,13 @@ impl<'lexer, 'input: 'lexer> JITIRParser<'lexer, 'input, '_> {
         self.process_func_types(func_types)?;
         self.process_func_decls(func_decls)?;
 
+        // We try and put trace inputs into registers, but place floating point values on the stack
+        // as yksmp currently doesn't seem able to differentiate general purpose from floating
+        // point.
```

Comment:
If we are indeed using DWARF reg numbers, a floating point reg is in the range `[17, 32]` inclusive, since we use the `xmm` vector registers.

Reply:
(that's x86_64 specific though!)